hw3/02-charts.js

```diff
@@ -1,52 +1,105 @@
 const backgroundColors = [
-  'rgba(54, 162, 235, 0.8)',
-  'rgba(255, 206, 86, 0.8)',
-  'rgba(255, 99, 132, 0.8)',
-  'rgba(75, 192, 192, 0.8)',
-  'rgba(153, 102, 255, 0.8)',
-  'rgba(255, 159, 64, 0.8)',
-  'rgba(199, 199, 199, 0.8)',
-  'rgba(83, 102, 255, 0.8)',
-  'rgba(40, 159, 64, 0.8)',
-  'rgba(210, 199, 199, 0.8)',
-  'rgba(78, 52, 199, 0.8)',
+	"rgba(54, 162, 235, 0.8)",
+	"rgba(255, 206, 86, 0.8)",
+	"rgba(255, 99, 132, 0.8)",
+	"rgba(75, 192, 192, 0.8)",
+	"rgba(153, 102, 255, 0.8)",
+	"rgba(255, 159, 64, 0.8)",
+	"rgba(199, 199, 199, 0.8)",
+	"rgba(83, 102, 255, 0.8)",
+	"rgba(40, 159, 64, 0.8)",
+	"rgba(210, 199, 199, 0.8)",
+	"rgba(78, 52, 199, 0.8)",
 ];
 
 const borderColors = [
-  'rgba(54, 162, 235, 1)',
-  'rgba(255, 206, 86, 1)',
-  'rgba(255, 99, 132, 1)',
-  'rgba(75, 192, 192, 1)',
-  'rgba(153, 102, 255, 1)',
-  'rgba(255, 159, 64, 1)',
-  'rgba(159, 159, 159, 1)',
-  'rgba(83, 102, 255, 1)',
-  'rgba(40, 159, 64, 1)',
-  'rgba(210, 199, 199, 1)',
-  'rgba(78, 52, 199, 1)',
+	"rgba(54, 162, 235, 1)",
+	"rgba(255, 206, 86, 1)",
+	"rgba(255, 99, 132, 1)",
+	"rgba(75, 192, 192, 1)",
+	"rgba(153, 102, 255, 1)",
+	"rgba(255, 159, 64, 1)",
+	"rgba(159, 159, 159, 1)",
+	"rgba(83, 102, 255, 1)",
+	"rgba(40, 159, 64, 1)",
+	"rgba(210, 199, 199, 1)",
+	"rgba(78, 52, 199, 1)",
```

Comment:
These colors could possible be placed in their own file that gets referenced for organization purpose

Reply:
Got it, I was even trying to fetch all the data together, and create another JSX component just to display that data. Maybe I will try this over a weekend.